controllers/harborconfiguration_controller.go

```diff
@@ -390,20 +390,23 @@ func getConcreteHarborType(ctx context.Context, crdClient dynamic.ResourceInterf
 }
 
 func deleteAll(ctx context.Context, harborConfiguration harborconfigurationv1alpha1.HarborConfiguration, client *apiv2.RESTClient) (ctrl.Result, error) {
-	err := client.DeleteRegistryByID(ctx, harborConfiguration.Status.RegistryId)
+	deletionLog := ctrl.Log.WithName("deletion")
```

Comment:
Why do we want a named logger here? Just for my understanding

Reply:
No reason in particular. I just copied what we had in main as a means of logging.